tardis_kromer_plot.py

```diff
@@ -356,6 +423,7 @@ def generate_plot(
         ylim=None,
         nelements=None,
         twinx=False,
+        species_list=None,
```

Comment:
I should have done this when I added the nelements functionality, but can you add a doc string thing here for it too?